src/layout/pushClient/Banner.jsx

```diff
@@ -1,24 +1,34 @@
 /* global localStorage */
 
-import styles from '../styles/buttonclient'
+import styles from './pushClient'
 
 import { translate } from 'cozy-ui/react/I18n'
 
 import React, { Component } from 'react'
+import { getTracker } from 'cozy-ui/react/helpers/tracker'
 
 import classNames from 'classnames'
 
-class ButtonClientMobile extends Component {
+const track = (element) => {
+  const tracker = getTracker()
+  if (!tracker) {
+    return
+  }
+  tracker.push(['trackEvent', 'interaction', 'desktop-prompt', element])
+}
+
+class BannerClient extends Component {
   constructor (props) {
     super(props)
     this.state = {
       seen: JSON.parse(localStorage.getItem('app_ad')) || false
```

Comment:
`app_ad` is used several times, in several files, and they are all linked — if you change one of the strings, the system breaks. I think it would really be worth creating a constant somewhere to hold this key.